helpdesk_mgmt/tests/common.py

```diff
@@ -33,6 +33,7 @@ def setUpClass(cls):
             context=ctx,
         )
         cls.stage_closed = cls.env.ref("helpdesk_mgmt.helpdesk_ticket_stage_done")
+        cls.channel_email = cls.env.ref("helpdesk_mgmt.helpdesk_ticket_channel_email")
```

Comment:
No need to put this in the base. It's not adding too much value for the rest of the tests.